src/components/Section.js

```diff
@@ -7,23 +7,35 @@ import { Row, Col } from 'react-bootstrap'
 
 export default class Section extends Component {
   static propTypes = {
-    actionButton: PropTypes.element
+    actionButton: PropTypes.element,
+    children: PropTypes.element,
+    name: PropTypes.string
   }
 
   render() {
     const { name, actionButton, children } = this.props
     return (
-      <div>
-        <Row className="section-header">
-          <Col sm={4}>
+      <>
+        <Row className="section-header hidden-md hidden-lg hidden-sm">
+          <Col xs={12}>
```

Comment:
I think that this kind of change could have an impact to the actual layout. Could you share a screen of both the views (desktop and mobile) where this change is applied?

Reply:
From this view looks like you should fix also the Maven icon.